pkg/opensearch/lucene_handler.go

```diff
@@ -132,20 +133,6 @@ func processLogsQuery(q *Query, b *client.SearchRequestBuilder, from, to int64,
 		size = defaultLogsSize
 	}
 	b.Size(size)
-
```

Comment:
Don't we need this to get the correct data back?

Reply:
I believe I started messing with this because I never saw my entire data.
I keep seeing only this:
![image](https://github.com/user-attachments/assets/fca468f3-08e5-4766-880d-a5dbcfd55b81)